tools/db_bench_tool.cc

```diff
@@ -4437,7 +4441,7 @@ class Benchmark {
         exit(1);
       }
       options.rate_limiter.reset(NewGenericRateLimiter(
-          FLAGS_rate_limiter_bytes_per_sec, 100 * 1000 /* refill_period_us */,
+          FLAGS_rate_limiter_bytes_per_sec, FLAGS_rate_limiter_refill_period_us,
```

Comment:
I didn't even think of we can do this when I left the comment - a new thing learnt! 👍